WebRoot/WEB-INF/tags/infra/view.tag

```diff
@@ -519,7 +519,7 @@ if (screen.width<=800) {
 								<td valign="top" colspan="3"></td>
 				<c:if test="${adsOn}" >
 								<td valign="top" style="border-top:1px solid #98ADBE; width:180px;">
-									<app:ads content="${ads}"/>
+									<app:ads content="${fn:escapeXml(ads)}"/>
```

Comment:
we usually prefer doing this sanitisation at leaf node, as there could be possibility that `<app:ads />` tag can be used from some other place also and developer missed to sanitise value before passing in the tag
so is it possible that we do this sanitisation inside the ads tag?

Reply:
@silentsakky after all findings I found that we have to use fn:escapeXml here and one more file that is ads.tag and I made changes. In infra/view.tag we have to use because it renders HTML in the ads sidebar and that HTML load alert dialog .